### ISSUE TITLE
Build GrantShares documentation

### ISSUE THREAD

**Issue by csmuller** [AxLabs/grantshares-docs#1]
- [x] #4
- [x] #5
- [x] #7
- [x] Add the contract hashes and links of the contracts on the main grantshares repo readme
- [ ] #6
- [x] #9
- [ ] #10
- [ ] #11
- [x] #12
- [ ] #23

**Comment by gsmachado** [comment]
We need to discuss this one.

Will we retire the Notion page and migrate everything to Docusaurus?
Or will we keep both, synced?

I would prefer to centralize things in a single place (for maintainability).

What do you think?

**Comment by gsmachado** [comment]
@csmuller Should we remove this issue from sprints and add the tasks (as issues)?!

**Comment by csmuller** [comment]
Yes, the tasks have to be more granular. I will come up with sub tasks

**Comment by gsmachado** [comment]
@OT-kraftchain check this issue.

I think we should discuss this and use AI wherever we can.

Back then, Claude and I didn't have AI to help us to write all the stuff. Things changed. We're way better equipped to generate documentation that makes sense without so much effort.

What do you think?
